code/modules/ruins/spaceruin_code/clericsden.dm

```diff
@@ -34,4 +34,13 @@
 
 /mob/living/simple_animal/hostile/construct/proteon/hostile //Style of mob spawned by trapped cult runes in the cleric ruin.
 	AIStatus = AI_ON
-	environment_smash = ENVIRONMENT_SMASH_STRUCTURES //standard ai construct behavior, breaks things if it wants, but not walls.
\ No newline at end of file
+	environment_smash = ENVIRONMENT_SMASH_STRUCTURES //standard ai construct behavior, breaks things if it wants, but not walls.
+
+/turf/open/floor/carpet/airless //atmos runtimes and all that goodness.
+	initial_gas_mix = AIRLESS_ATMOS
+
+/turf/open/floor/carpet/royalblack/airless
+	initial_gas_mix = AIRLESS_ATMOS
+
+/turf/open/floor/carpet/royalblue/airless
+	initial_gas_mix = AIRLESS_ATMOS
```

Comment:
just complete the set while you're here for future mapping

Reply:
Good to go.